test/acceptance/features/steps/steps.py

```diff
@@ -303,27 +304,55 @@ def quarkus_app_is_imported_as_knative_service(context, application_name):
     context.application_type = "knative"
 
 
-# STEP
+def check_result(secret_name):
+    check = secret_name is not None
+    return check
+
+
+def get_sbr_secret_name(context):
+    openshift = Openshift()
+    output = ""
+    if "sbr_name" in context:
+        json_path = "{.status.secret}"
+        output = openshift.get_resource_info_by_jsonpath("servicebindings", context.sbr_name, context.namespace.name, json_path)
+    assert output is not None, "Failed to fetch secret name from ServiceBinding"
+    return output
+
+
+def check_secret_update(context):
```

Comment:
please rename this function to reflect what is happening inside it. I can see following things:
* get actual secret name
* compare it to what is saved in the context
* if equal return None (why?)
* otherwise overwrite the secret name in the context and return the new name.

By looking at the function usage (line 322), it looks that the function is not really needed because pooling with wait until service binding gets a secret assigned to it.  Hence, `then_secret_should_be_updated` can be rewritten as following:

```python
def then_secret_should_be_updated(context):
  context.sb_secret = polling2.poll(lambda: get_sbr_secret_name(context), step=100, timeout=1000, ignore_exceptions=(ValueError,), check_success=lambda v: v is not None and context.sb_secret != v)
```

with that `check_secret_update` can be removed.

Reply:
Done